GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle more contributions, from a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package,
+which is guided by the following principles:
+
+* Few (if any) dependencies
+* Time & memory efficiency
+* Concise syntax (minimal redundancy in code)
+* Stable code base (easy for users to upgrade to new data.table, and compatible with old R versions)
+* Comprehensive and accessible documentation
+* Clear error and warning messages
+
+To prioritize developer time, we define what is in and out of scope.
+The current scope of package functionality includes:
+* Data manipulation and analysis 
+    * reshaping/pivoting
+    * aggregation/summarizing
+    * subsetting rows
```

Comment:
feels needlessly un-parallel for rows vs. columns

```suggestion
    * adding/updating/deleting rows
```

Reply:
I thought that would be inefficient?